test/api/graph.js

```diff
@@ -0,0 +1,70 @@
+var assert = require("assert");
```

Comment:
No worries, as you can see from the code, tests are a huge missing piece, thought I'd take an opportunity here.